### ISSUE TITLE
Player can still move when ragdolled

### ISSUE THREAD

**Issue by EddieSoloProjects** [Dixon6/GMC-ALS#9]
It's a minor thing, But a Player can still move when ragdolled. The W, A, S, D keys are still active.

**Comment by Dixon6** [comment]
Yeah noticed that one as well. Seems like halt movement isn't enough on the premovement tick to stop completely. Will need to check around a bit. Still unhappy with Ragdoll

**Comment by EddieSoloProjects** [comment]
I'm thinking disconnect the Input on the keys until Unragdolled

**Comment by Dixon6** [comment]
Was getting some other errors when trying that. Once I recheck ragdoll I'll take a look

**Comment by h4sohail** [comment]
Any updates on this?
